Makefile

```diff
@@ -19,6 +19,11 @@ test_playground:
 	done;
 
 proto:
+	@pv=`protoc --version`; \
```

Comment:
This is already done in the proto.sh file, no ?

Reply:
Ok I reached the change in proto.sh. Is it because Travis uses a different version and you can't install the 3.6.1 one ?

Reply:
`protoc` is not installed at all in travis, so I would've to make sure that it is installed in the correct version. And in fact `protoc` is never used in `proto.sh` at all, only in `external/Makefile`. So it should go in there.

Reply:
Put check of protoc-version in `external/Makefile`.